module-1/src/App.tsx

```diff
@@ -48,49 +47,85 @@ function App() {
 
   const handleSortByChange = (value: string) => {
     console.log("SortBy selected:", value);
-    setSelectedSort(value);
+  };
+
+  const handleCloseDialog = () => {
+    console.log("CLOSING MODAL");
+    setOpenDialog(false);
+  };
+
+  const handleDialogSubmit = () => {
+    console.log("SUBMIT MODAL");
+    setOpenDialog(false);
+  };
+
+  const handleCloseDialogEdit = () => {
+    console.log("CLOSING EDIT MODAL");
+    setOpenDialogEdit(false);
+  };
+
+  const handleDialogSubmitEdit = () => {
+    console.log("SUBMIT EDIT MODAL");
+    setOpenDialogEdit(false);
   };
 
   return (
-    <div className="container mx-auto p-3">
-      <h1 className="font-bold text-2xl  m-2 text-center ">
-        {" "}
+    <div className="container mx-auto p-3 bg-custom_dark_gray">
+      <h1 className="text-2xl  m-2 text-center text-white font-Montserrat">
         ReactJS Mentor Program
       </h1>
-      <div className="m-5 text-center">
-        <Counter initialValue={initialValue}></Counter>
-      </div>
 
-      <div className="m-5 text-center">
+      <div className="banner flex flex-col justify-between m-5 text-center">
+        <div className="flex justify-end">
+          <button
+            type="button"
+            className="bg-custom_light_gray/70 rounded-lg text-custom_pink my-1 mx-1 py-1 px-1 hover:bg-custom_light_gray/90"
+            onClick={() => setOpenDialog(true)}
+          >
+            +Add Movie
+          </button>
+          <button
+            type="button"
+            className="bg-custom_light_gray/70 rounded-lg text-custom_pink my-1 mx-1 py-1 px-1 hover:bg-custom_light_gray/90"
+            onClick={() => setOpenDialogEdit(true)}
+          >
+            +Edit Movie
+          </button>
+        </div>
         <Search initialSearchQuery="initial search" onSearch={handleSearch} />
-        <p className="bg-slate-400 m-4">{searchResults} </p>
+        <div className="flex justify-between items-center">
+          <Genres
+            genres={listOfGenres}
+            selected={selectedGenre}
+            onSelect={handleGenreSelection}
+          ></Genres>
+          <SortControl
+            sortByDefaultValue="Title"
+            sortByValues={sortValues}
+            onChanges={handleSortByChange}
+          ></SortControl>
```

Comment:
You can use shorthand notation.
```suggestion
          <Genres
            genres={listOfGenres}
            selected={selectedGenre}
            onSelect={handleGenreSelection}
          />
          <SortControl
            sortByDefaultValue="Title"
            sortByValues={sortValues}
            onChanges={handleSortByChange}
          />
```